src/main/webapp/app/stores/firebase/firebase-app.store.ts

```diff
@@ -52,15 +52,19 @@ export class FirebaseAppStore extends BaseStore {
   }
 
   initializeFirebase() {
-    const { enabled, ...firebaseOptions } = AppConfig.serverConfig.frontend.firebase;
+    const { enabled, isTest, ...firebaseOptions } = AppConfig.serverConfig.frontend.firebase;
     this.firebaseEnabled = enabled;
     if (this.firebaseEnabled) {
       try {
         this.firebaseOptions = firebaseOptions as FirebaseOptions;
         this.firebaseApp = initializeApp(this.firebaseOptions);
         this.firebaseDb = getDatabase(this.firebaseApp);
-        const auth = getAuth();
         this.firebaseInitSuccess = true;
+
+        const auth = getAuth();
+        if (isTest) {
+          connectAuthEmulator(auth, 'http://127.0.0.1:9099', { disableWarnings: true });
```

Comment:
Is it possible to read `firebase.json` to get the port. If not, do you think we can make the URL a constant

Reply:
We can't read it in since `firebase.json` is not included in the webpack build, so I will create a constant